### PR TITLE
Web platform highlights for Edge 91

### PR DESCRIPTION
**Pull request by KeldaAnders**
Publish first round of Web platform highlights (Edge 91)

### CODE REVIEW

File: microsoft-edge/web-platform/whats-new/edge-web-platform-91.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<!--
+description: These highlights focus on notable, developer-consumable updates to the web platform. We are continually working on exciting projects, fundamental improvements, and consumer-facing platform changes and are delighted to share what we are releasing in Edge.
+title: What's New to Web Platform (Microsoft Edge 92)
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/whats-new/edge-web-platform-91.md:3]: # h1 says 91

File: microsoft-edge/web-platform/whats-new/edge-web-platform-91.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<!--
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/whats-new/edge-web-platform-91.md:1]: Remove HTML comment tagging to reveal the YAML fields and make build errors go away.

File: microsoft-edge/web-platform/whats-new/edge-web-platform-91.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+[Import assertions](https://github.com/tc39/proposal-import-assertions)  and [JSON modules](https://github.com/tc39/proposal-json-modules). These allow developers to import JSON data, such as localized strings, into their JavaScript environments via the same ES module dependency system for JavaScript dependencies. 
+
+
+## Site Compatibility Changes
+
+<!-- Title: Site Compatibility Changes -->  
+
+*   For standards-compliance, [a change was made for Edge 91](https://chromium.googlesource.com/chromium/src/+/b332c5609ed9caa5979c24a0d1bdea242329258c) to the handling of SameSite=Lax cookies to prevent them from being sent on cross-site subframe navigations. If this change breaks a scenario, the cookie in question will need to be marked SameSite=None. Note: Chrome shipped this change in Chrome v90 while Edge is shipping in v91. 
+
+*   See more impacting changes that are coming to Microsoft Edge, [here](https://docs.microsoft.com/en-us/microsoft-edge/web-platform/site-impacting-changes).
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/whats-new/edge-web-platform-91.md:27]: Global: delete "/en-us" from Microsoft URLs, to make build errors/warnings go away.

Comment (mikehoffms) [reply microsoft-edge/web-platform/whats-new/edge-web-platform-91.md:27]: Global: any URLs that start with 
https://docs.microsoft.com/en-us/microsoft-edge/
or
https://docs.microsoft.com/microsoft-edge/
are in the present docs repo and should be changed to relative paths, to make build errors go away.

File: microsoft-edge/web-platform/whats-new/edge-web-platform-91.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+*   [Accessible Confirmation of Action](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/Accessibility/ConfirmationOfAction/explainer.md) explainer has been released and proposes a notification API that better solves for use cases where ARIA live regions aren't desirable.  
+
+*   We published [MaCAW](https://github.com/WICG/privacy-preserving-ads/blob/main/MACAW.md), which builds on [Parakeet](https://github.com/WICG/privacy-preserving-ads/blob/main/Parakeet.md), Microsoft’s WICG proposal for privacy preserving web monetization model for post 3P cookies deprecation. 
+
+*   Microsoft began participating in the following W3C standards groups:   
+
+    *   [Machine Learning Working Group](https://www.w3.org/community/webmachinelearning) (Supporting the WebNN specification),
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/whats-new/edge-web-platform-91.md:41]: Remove commas from these list items.

**Comment by vchapel** [comment]
I did a casual read-through and didn't see anything you haven't already called out. If you want me to do a more thorough reading, add me as a reviewer whenever you want me to do so.

**Pull request closed without merging**